### ISSUE TITLE
Warning: is Mounted(...) is deprecated

### ISSUE THREAD

**Issue by suhabaobaid** [oblador/react-native-collapsible#167]
I am having this warning isMounted(...) is deprecated in plain Javascript classes. Instead, make sure to clean up subscriptions and pending requests in componentWillUnmount to prevent memory leaks. 
This warning shows up when transitioning from collapsed to non-collapsed. I tried to find where isMounted is used in the code but with no luck. 

package.json:
"react": "16.3.1",
"react-native": "0.55.2"

**Comment by iRoachie** [comment]
It’s a generic error that is being displayed since we’re on a beta version of react 16.3 which deprecates some lifecycle methods. https://github.com/facebook/react-native/issues/18175

**Comment by suhabaobaid** [comment]
Oh ok thanks for the quick reply. Seems they are still fixing stuff over at React.

**Comment by ericketts** [comment]
It's still deprecated, unless you've got another ticket regarding migrating away from the deprecated lifecycle methods, it seems like a good idea to keep this one open.

**Comment by iRoachie** [comment]
Already a ticket open @ericketts https://github.com/oblador/react-native-collapsible/issues/162

**Comment by iRoachie** [comment]
Lol Hope you can sleep after

**Comment by Dihan2715** [comment]
Still waiting on this to be fixed. It keeps my iMac super laggy because it produce memory leaks every time a component is mounting or unmounting. My development is alwasy interrupted so I need to restart the simulator. Btw I'm using Expo on my case.

**Comment by enrtama** [comment]
+1, same issue
